dailystats.py

```diff
@@ -36,17 +40,19 @@ def read_stats_file(csv_file_name):
         new_line_value_list = [(field_list[3], field_list[4])]
         # if field_list[0] != "England" and field_list[0] != "Wiltshire":
         #     continue
-        if DEBUG:
-            print("Region is ==> {}".format(field_list[0]))
+        LOGGER.debug(
+            "Region is ==> {}".format(field_list[0])
+        )
         existing_value_list = the_dictionary.get(field_list[0])
         if existing_value_list is None:
             the_dictionary[field_list[0]] = new_line_value_list
         else:
             existing_value_list.append(new_line_value_list)
             the_dictionary[field_list[0]] = existing_value_list
-    if DEBUG:
-        print("\n\n Input File has been processed. Line count is {}".format(linecount))
-        print("Dict for region ==>  {} {} ".format("England", the_dictionary.get("England")))
+    LOGGER.debug(
+        f"\n\nInput File has been processed. Line count is {linecount}"
+        f"\nDict for region ==>  England {the_dictionary.get('England')}"
+    )

```

Comment:
the standard `logging` library already caters for log levels - this removes the need for all of the `if` statements